### DISCUSSION TITLE
Webhook unavailable 503

### DISCUSSION THREAD

**Discussion by Davnov1** [AntlerVC/firetable#1416] [Support / Q&A] [answered]
Hello,

I am experiencing an issue with my payment gateway webhook integration. Whenever a payment is successfully processed, it's supposed to send a POST request to my webhook hosted on Google Cloud Run. However, I am consistently receiving a 503 Service Unavailable error. The failure reason is indicated as ILLEGAL_HTTP_CODE.

Here are some relevant logs:

yaml
Copy code
Sep 17, 2023 @ 16:59:04.649560012 
failed(): failureReason=ILLEGAL_HTTP_CODE, httpCode=503

Sep 17, 2023 @ 16:59:04.649609473 
java.lang.RuntimeException: Failed to process notification to host, failureReason=illegal_http_code, httpCode=503
The webhook is designed to handle payment notifications and is crucial for the functioning of my application. I've tried troubleshooting this on my end but have been unable to pinpoint the cause of the issue.

Could you please help me identify what might be causing the 503 error and guide me on how to resolve it? I'm keen to understand whether this is a problem with my server configuration, an issue with YooKassa's service, or something else entirely.

Thank you for your time and assistance

**Comment by Chadha93** [comment]
Hi @Davnov1, thank you for sharing the logs. Could you please share a jam.dev video and your project ID with your email?

You might want to try [BuildShip](https://buildship.com/) to implement this.

**Reply by Chadha93** [reply, marked as answer]
Closing this issue for now, feel free to reopen with more information.
